types/vue.d.ts

```diff
@@ -50,7 +50,9 @@ declare namespace NuxtVueI18n {
       routesNameSeparator: string
       beforeLanguageSwitch: () => any
       onLanguageSwitched: () => any
+      getLocaleCookie: () => null | string
```

Comment:
Should return `string | undefined | null` since js-coookie can return `undefined` and our method here can return `null`.

js-cookie: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/47f0eec500962b5beec44f18b7ae33b843c5d699/types/js-cookie/index.d.ts#L51
cookie: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/47f0eec500962b5beec44f18b7ae33b843c5d699/types/cookie/index.d.ts#L113